src/Libraries/DynamoPython/Python.csproj

```diff
@@ -1,52 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ImportGroup Label="PropertySheets">
-    <Import Project="$(SolutionDir)Config\CS.props" />
-  </ImportGroup>
-  <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
hmm - I think this is another one of those projects we should have gotten rid of at 2.13 and we're stuck with it for another year?

Reply:
I can add an empty file for it.

Reply:
LOoks like this project was not in the solution. SO we should be able to remove it

Reply:
nice, thanks for confirmig.